website/source/layouts/docs.erb

```diff
@@ -268,8 +268,8 @@
             <a href="/docs/auth/approle.html">AppRole</a>
           </li>
 
-          <li<%= sidebar_current("docs-auth-aws-ec2") %>>
-            <a href="/docs/auth/aws-ec2.html">AWS EC2</a>
+          <li<%= sidebar_current("docs-auth-aws") %>>
+            <a href="/docs/auth/aws-ec2.html">AWS</a>
```

Comment:
I'm asking internally but I believe this is the right way, yeah. Thanks for renaming it.